examples/kyb-app/src/components/organisms/UIRenderer/elements/Cell/Cell.tsx

```diff
@@ -0,0 +1,52 @@
+import { CSSProperties, useMemo } from 'react';
+import chunk from 'lodash/chunk';
+import { BlocksComponent } from '@ballerine/blocks';
+import { Column } from '@app/components/organisms/UIRenderer/elements/Cell/Column';
+import { useUIRendererContext } from '@app/components/organisms/UIRenderer/hooks/useUIRendererContext/useUIRendererContext';
+import clsx from 'clsx';
+import { Fragment } from 'react';
+
+export interface CellOptions {
+  columns?: number;
+  styles?: CSSProperties;
+  className?: string;
+  align?: 'left' | 'right';
+}
+
+export interface CellProps {
+  options: CellOptions;
+  childrens: any[];
+}
+
+export const Cell = ({ options = {}, childrens: _childrens = [] }: CellProps) => {
```

Comment:
What do you think about `blocks` over `_childrens`? `_` is untrue since the variable is in use.

Reply:
Sounds good.